cmd/apps/nfs_app.go

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) arkade author(s) 2020. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package apps
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path"
+
+	"github.com/alexellis/arkade/pkg"
+	"github.com/alexellis/arkade/pkg/apps"
+	"github.com/alexellis/arkade/pkg/config"
+	"github.com/alexellis/arkade/pkg/env"
+	"github.com/alexellis/arkade/pkg/helm"
+	"github.com/alexellis/arkade/pkg/types"
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallNfsProvisioner() *cobra.Command {
+	var nfsProvisionerApp = &cobra.Command{
+		Use:          "nfs-client-provisioner",
+		Short:        "Install nfs client provisioner",
+		Long:         "Install nfs client provisioner to create dynamic persistent volumes",
+		Example:      "arkade install nfs-client-provisioner --nfs-server=x.x.x.x --nfs-path=/exported/path",
+		SilenceUsage: true,
+	}
+
+	nfsProvisionerApp.Flags().StringP("namespace", "n", "default", "The namespace to install nfs-client (default: default")
+	nfsProvisionerApp.Flags().String("nfs-server", "", "")
```

Comment:
can we have a description for these `nfs-server` and `nfs-path` flags - these will get printed when someone uses the help command.

Reply:
👍

Reply:
This was on my list of things to fix before removing the WIP